### PR TITLE
Feedback request

### PR DESCRIPTION
**Pull request by KarenJoo**
Not sure if I'm doing this properly, but I am asking for feedback if possible before delivery. My design is obviously not done, but I am focused on complying with the marking criteria for the CA and would prefer feedback based on that. Am I extending Bootstrap and .scss correctly? how is my .scss according to the DRY principle? and if time to review, what is the overall impression of the design/app?

### CODE REVIEW

File: authentication/index.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  
+  <body class="body-login">
+    <nav class="navbar navbar-expand-lg navbar-custom pt-5 pb-5">
+      <img class="logo" src="/images/logo.png" alt="logo of Looking for Frieda">
+    </nav>
+    
+    <!-- form validation -->
+    <div class="container">
+      <div class="row justify-content-center align-items-center min-vh-100">
+        <div class="col-12col-md-8">
```
Comment (Fermain) [inline authentication/index.html]: There is a missing space here, check out offsets: https://getbootstrap.com/docs/5.3/layout/columns/#offsetting-columns

File: authentication/index.html [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+                </button>
+              </form>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.slim.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.1/umd/popper.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.6.0/js/bootstrap.min.js"></script>
```
Comment (Fermain) [inline authentication/index.html]: If you swap this for bootstrap.bundle, you only need one script tag:

`    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/js/bootstrap.bundle.min.js" integrity="sha384-HwwvtgBNo3bZJJLYd8oVXjrBZt8cqVSpeBNS5n7C8IVInixGAoxmnlMuBnhbgrkm" crossorigin="anonymous"></script>`

File: src/scss/index.scss [hunk 3]
```diff
@@ -1 +1,154 @@
+$primary: #67927a;
+$secondary: #eae6e5;
+$dark-color: #000;
+$secondary-light: #eae6e5c4;
+$primary-light: #67927ae9;
+$box-shadow: 2px 2px 4px rgba(0, 0, 0, 0.16);
```
Comment (Fermain) [inline src/scss/index.scss:7]: You are not importing Bootstrap!

`@import "../../node_modules/bootstrap/scss/bootstrap";`

File: src/scss/index.scss [hunk 4]
```diff
@@ -1 +1,154 @@
...
+
+.logo {
+  width: 100%; /* Adjust as needed */
+  max-width: 400px; /* Maximum width for the logo */
+  display: block;
+  margin: 0 auto;
+}
+
+/* Log in form */
+.form-container {
```
Comment (Fermain) [inline src/scss/index.scss:57]: There are too many custom styles here, background colours, border radius, padding, flex, alignment can all be done with Bootstrap styles!

File: profile/index.html [hunk 5]
```diff
@@ -0,0 +1,178 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <meta name="Looking for Frieda | Profile" content="Looking for Frieda profile page" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.6.0/css/bootstrap.min.css"
```
Comment (Fermain) [inline profile/index.html:10]: If you get the SCSS setup correctly, you won't need this

File: package.json [hunk 6]
```diff
@@ -22,5 +24,11 @@
   "bugs": {
     "url": "https://github.com/NoroffFEU/css-frameworks-ca/issues"
   },
-  "homepage": "https://github.com/NoroffFEU/css-frameworks-ca#readme"
+  "homepage": "https://github.com/NoroffFEU/css-frameworks-ca#readme",
+  "devDependencies": {
+    "bootstrap": "^5.2.0",
```
Comment (Fermain) [inline package.json:29]: Bootstrap should be a dependency, not a devDependency:

`npm uninstall bootstrap`
`npm install bootstrap`

File: package.json [hunk 7]
```diff
@@ -22,5 +24,11 @@
...
+    "live-server": "^1.2.2",
+    "node-sass": "^6.0.1",
```
Comment (Fermain) [inline package.json:31]: You can uninstall node-sass:

`npm uninstall node-sass`

File: package.json [hunk 8]
```diff
@@ -3,7 +3,9 @@
   "version": "0.0.1",
   "description": "A starting point for the CSS Frameworks CA project",
```
Comment (Fermain) [inline package.json:4]: Please update your package description to describe what you have created

**Review by Fermain — CHANGES_REQUESTED** [review]
The most important change is getting Bootstrap SCSS imported correctly, and reducing the number of custom styles you are using.

File: package.json (hunk 6 above)
Comment (KarenJoo) [reply package.json:29]: started all over again: bootstrap is now a dependency

File: src/scss/index.scss (hunk 3 above)
Comment (KarenJoo) [reply src/scss/index.scss:7]: started all over again: bootstrap is now correctly installed

File: src/scss/index.scss (hunk 4 above)
Comment (KarenJoo) [reply src/scss/index.scss:57]: started all over again: thanks, I have misunderstood bootstrap and am now deleting custom styling and replacing it with bootstrap styles

File: profile/index.html (hunk 5 above)
Comment (KarenJoo) [reply profile/index.html:10]: started all over again: thanks

File: package.json (hunk 7 above)
Comment (KarenJoo) [reply package.json:31]: started all over again: node-sass is not installed this time and is not to befound under devDependencies

File: package.json (hunk 8 above)
Comment (KarenJoo) [reply package.json:4]: started all over again: description updated

**Pull request merged**